maturity-model.md

```diff
@@ -9,10 +9,10 @@ Tier | Content
 0 | IDC Minimalization > Project Formation using Socio-Economic Lens & #DifferentTogether Pledge
 1 | Test product/service for minorities: BIPOC, LGBTQIA2S+, Non-Binary, People with Disabilities, Residents in Rural areas
 CI  | Check in - Substantial analysis completed for all groups?
-2 | IDC Acceptance > Intersect product/service analysis with privacy and security
-CI | Check in - 
-3 | Apply Privacy and Security
+2 | IDC Acceptance > Intersect product/service analysis with privacy and security components, network infrastructure and both semantics and syntax of language

```

Comment:
Can acronyms like IDC please be listed somewhere?

Reply:
I do have a sources page and I will link it to the acronym after the model.